torchaudio/prototype/ctc_decoder/ctc_decoder.py

```diff
@@ -126,16 +126,24 @@ def __call__(self, emissions: torch.FloatTensor, lengths: Optional[torch.Tensor]
             List[List[torchaudio.prototype.ctc_decoder.Hypothesis]]
 
         Args:
-            emissions (torch.FloatTensor): tensor of shape `(batch, frame, num_tokens)` storing sequences of
-                probability distribution over labels; output of acoustic model
-            lengths (Tensor or None, optional): tensor of shape `(batch, )` storing the valid length of
-                in time axis of the output Tensor in each batch
+            emissions (torch.FloatTensor): CPU tensor of shape `(batch, frame, num_tokens)` storing sequences of
+                probability distribution over labels; output of acoustic model.
+            lengths (Tensor or None, optional): CPU tensor of shape `(batch, )` storing the valid length of
+                in time axis of the output Tensor in each batch.
 
         Returns:
             List[List[Hypothesis]]:
                 List of sorted best hypotheses for each audio sequence in the batch.
         """
-        assert emissions.dtype == torch.float32
+
+        if emissions.dtype != torch.float32:
+            raise ValueError("emissions must be float32.")
+
+        if emissions.is_cuda:
+            raise RuntimeError("emissions must live on CPU.")
```

Comment:
similar nit as above

```suggestion
            raise RuntimeError("emissions must be a CPU tensor.")
```